aws-lc-sys/builder/cc_builder/aarch64_apple_darwin.rs

```diff
@@ -1,6 +1,6 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0 OR ISC
-// Thu Aug  1 23:56:17 UTC 2024
+// Wed Aug 21 14:33:51 UTC 2024
```

Comment:
This isn't a typical diff.  We normally only regenerate the bindings when the version number changes (which it will soon). 

When the version number changes, nearly every definition included in the bindings is modified due to the version number being part of the symbol for linkage. In this PR, this diff simply confirms that these bindings were also re-generated (otherwise unmodified) during the workflow run.